templates/blocks/p4_structured_data.twig

```diff
@@ -0,0 +1,8 @@
+{% set social_menu_links = [] %}
+{% if footer_social_menu %}
+	{% for menu_details in footer_social_menu %}
+		{% set social_menu_links = social_menu_links|merge([menu_details.url]) %}
+	{% endfor %}
+{% endif %}
+<script type='application/ld+json'>{"@context":"http:\/\/schema.org","@type":"WebSite","url":"{{ site.link }}","name":"{{ site.name }}","alternateName":"Greenpeace","potentialAction":{"@type":"SearchAction","target":"{{ site.link }}/?s={search_term_string}&orderby=_score","query-input":"required name=search_term_string"}}</script>
```

Comment:
Backslashes are not needed here, `http://schema.org`

Reply:
Since you want to parse Json, I would prefer if you created your json in a more structured and less error prone way, like: 
```
$arr = [
    '@context' => 'http://schema.org', 
    '@type' => 'Website', 
    'url' => site.link, 
    'd' => 4, '
    e' => 5
];

$myjson = json_encode($arr);
```

Reply:
The previous commit is amended. Please have a look.